source/extensions/access_loggers/grpc/BUILD

```diff
@@ -51,6 +51,7 @@ envoy_cc_library(
         "//source/common/network:utility_lib",
         "//source/common/stream_info:stream_info_lib",
         "//source/common/stream_info:utility_lib",
+        "//source/common/tracing:http_tracer_lib",
```

Comment:
I don't think gRPC access logger should pull whole HTTP tracer as a dependency. Can you pull `CustomTagBase` and related class to different file and an `envoy_cc_library`?

Reply:
done